### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by serviza**

### CODE REVIEW

File: app/test_split_integer.py [hunk 1]
```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    parts = split_integer(17, 4)
+    assert sum(parts) == 17
```
Comment (Menselface) [inline app/test_split_integer.py:7]: Code Style
Don't create unnecessary variables for simple tests:
Good example:

def test_():
    assert some_function_to_test() == [10]
    
 Don't forget to check [checklist ](https://github.com/mate-academy/py-split-integer/blob/master/checklist.md)before you pushing your solution
 assert sum(split_integer(17, 4) == 17

File: app/test_split_integer.py [hunk 2]
```diff
@@ -2,20 +2,25 @@
...
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    parts = split_integer(16, 4)
+    assert parts[0] == parts[1] == parts[2] == parts[3]
```
Comment (Menselface) [inline app/test_split_integer.py:12]: Same here
assert split_integer(16, 4) == [4, 4, 4, 4]

File: app/test_split_integer.py [hunk 3]
```diff
@@ -2,20 +2,25 @@
...
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    parts = split_integer(13, 1)
+    assert parts[0] == 13
```
Comment (Menselface) [inline app/test_split_integer.py:17]: Same story

File: app/test_split_integer.py [hunk 4]
```diff
@@ -2,20 +2,25 @@
...
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    parts = split_integer(32, 5)
+    assert parts == sorted(parts)
```
Comment (Menselface) [inline app/test_split_integer.py:22]: +

File: app/test_split_integer.py [hunk 5]
```diff
@@ -2,20 +2,25 @@
...
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts():
-    pass
+    parts = split_integer(3, 5)
+    assert parts[0] == parts[1] == 0
```
Comment (Menselface) [inline app/test_split_integer.py:26]: end here

**Review by Menselface — CHANGES_REQUESTED** [review]

**Review by Menselface — APPROVED** [review]
Super

File: app/test_split_integer.py [hunk 6]
```diff
@@ -2,20 +2,21 @@
...
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(13, 1) == [13]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    parts = split_integer(32, 5)
+    assert parts == sorted(parts)
```
Comment (Y-Havryliv) [inline app/test_split_integer.py:18]: it's a bad test because `parts` is the result of `split_integer(32, 5)` and you're comparing `parts` to `parts`

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**